package-overrides/npm/angular-aria@1.5.0.json

```diff
@@ -0,0 +1,15 @@
+{
+  "format": "global",
+  "jspmNodeConversion": false,
+  "main": "angular-aria",
+  "dependencies": {
+    "angular": "npm:angular@^1.5.0"
```

Comment:
Ok makes sense. The main reason why I've switched to npm was that to avoid hitting github rate limits (though this should be no longer an issue) and that I can use our internal NPM registry for caching those external packages so that an installation works also "offline". If I rely on the Github repos I've somehow mirror these repos on demand which is quite complicated.

So for now I'll place these npm overwrites in my private jspm registry so that I can install all packages from the internal NPM registry.